src/app/objective/objective.component.css

```diff
@@ -9,3 +9,10 @@
 .assign-button-enabled a { color: #674ab7; text-decoration: none; }
 .assign-button-enabled a:hover { text-decoration: underline; }
 .assign-button-disabled { color: grey; }
+.sum-all { display: flex; justify-content: center;   width: 10%; align-items: center;} 
```

Comment:
I could be wrong, but I think we can merge the `sum-all` and `sum` classes. We should probably call it something more descriptive like `.resource-csum`.